build.md

```diff
@@ -301,6 +301,76 @@ build:
   target: prod
 ```
 
+### secrets
+`secrets` grants access to sensitive data defined by [secrets](secrets) on a per-service build basis. Two
+different syntax variants are supported: the short syntax and the long syntax.
+
+Compose implementations MUST report an error if the secret doesn't exist on the platform or isn't defined in the
```

Comment:
```suggestion
Compose implementations MUST report an error if the secret isn't defined in the
```
secret MUST be declared in compose file. One can use an existing secret on platform by defining an `external: true` secret